runtime/src/main/java/org/corfudb/runtime/view/LayoutManagementView.java

```diff
@@ -2,15 +2,17 @@
 
 import java.util.Collections;
 import java.util.Comparator;
+import java.util.List;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.List'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)